### PR TITLE
add unittest for using None as weight in loss function

### PR DESCRIPTION
**Pull request by ZhiYuan-REN**
As I see the change only happens in classfication_loss.py and regression_loss.py, so I add the corresponding test part in test_classification_loss.py and test_regression_losses.py.

### CODE REVIEW

File: tests/test_loss/test_classification_loss.py [hunk 1]
```diff
@@ -37,3 +45,8 @@ def test_bce_loss():
     assert torch.allclose(
         loss(fake_pred, fake_label, fake_weight),
         -torch.log(torch.tensor(0.5)))
+
+
+if __name__ == '__main__':
```
Comment (jin-s13) [inline tests/test_loss/test_classification_loss.py:50]: We do not need `__main__` in pytest

**Comment by jin-s13** [comment]
Please sign CLA, and check the linting

File: tests/test_loss/test_classification_loss.py (hunk 1 above)
Comment (ZhiYuan-REN) [reply tests/test_loss/test_classification_loss.py:50]: ok. I use this to test locally. I forget to delete it when I commit.

Comment (ly015) [reply tests/test_loss/test_classification_loss.py]: You can run pytest locally without __main__.

Comment (ZhiYuan-REN) [reply tests/test_loss/test_classification_loss.py]: ok!

**Comment by jin-s13** [comment]
.pre-commit-config.yaml  should not be deleted.

**Comment by ZhiYuan-REN** [comment]
> .pre-commit-config.yaml should not be deleted.

ok.My fault! I just don't hope the change in config.yml will be merged!

**Comment by ZhiYuan-REN** [comment]
Sry, bros, I need to relaunch a PR. Because I change something unrelated to my task. (Specifically, I thought mmpose don't have pre-commit-config.yml, so I write a pre-commit-config.yml and use it auto-fix sth.). So I hope to close this PR. Forgive me!
